src/management.mligo

```diff
@@ -13,7 +13,9 @@
 let transfer_ownership
     (param, store : transfer_ownership_param * storage) : return =
       let store = authorize_admin(store) in
-        (([] : operation list), { store with pending_owner = param ; })
+        if Tezos.self_address = param
+          then (([] : operation list), { store with admin = param ; }) // If new admin is address of baseDAO, set as admin right away.
+          else (([] : operation list), { store with pending_owner = param ; })
```

Comment:
A bit of a nitpick, but it should be slightly more efficient to use the ops list once, also the comment takes the line too far out:
```
      let store =
            if Tezos.self_address = param
            then
               // If new admin is address of baseDAO, set as admin right away.
              { store with admin = param ; }
            else
              { store with pending_owner = param ; }
      in (nil_op, store)
```

Reply:
Have updated.